### DISCUSSION TITLE
Plots Not Being Queued in Phase 1, Despite max_concurrent Not Being Reached

### DISCUSSION THREAD

**Discussion by ryanng3120** [swar/Swar-Chia-Plot-Manager#475] [Q&A]
Hey everyone, I am running into an issue where Swar does not queue new plots into phase 1. 

Here's what I see (refer to screenshots for more details): 
- 8 plots max at any given time
- no plots in phase 1 even though max_concurrent and max_for_phase_1 is not reached

Is there something in my config that's preventing plots from being kicked off into phase 1? 
Thanks in advance!

![Screenshot from 2021-05-17 15-34-01](https://user-images.githubusercontent.com/84187991/118459884-61a86c00-b726-11eb-899c-b63aa842fc78.png)
![Screenshot from 2021-05-17 15-34-13](https://user-images.githubusercontent.com/84187991/118459897-62d99900-b726-11eb-97ae-8fab4f81ac0f.png)

**Comment by Cminer3207** [comment]
Your max_concurrent is at 7, so why would it start a new plot when you already have 8 running? 
Be sure to check Global setting (above the individual job-order settings). 
max_con is a Global setting and applies to all jobs. It’s not a per-job setting.
